setup.py

```diff
@@ -29,20 +29,19 @@
 # 'Development Status :: 5 - Production/Stable'
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
+    "dataclasses >= 0.8, < 1.0; python_version < '3.7'",  # Until we drop Python 3.6 support.
```

Comment:
Do we still need `dataclasses`? I don't see it used anywehre.

Reply:
Nope, you're right. After refactoring the models again, `dataclasses` dependency is not needed anymore.